### PR TITLE
test(cosmic-swingset): fix and refactor make targets

### PR DESCRIPTION
**Pull request by samsiegart**

### CODE REVIEW

File: packages/cosmic-swingset/Makefile [hunk 1]
```diff
@@ -178,7 +178,7 @@ t1/$(BASE_PORT)/owner-address:
 	$(AGCH) --home=t1/$(BASE_PORT)/owner keys --keyring-backend=test add owner | jq -r .address > $@.tmp
 	mv $@.tmp $@
 
-# Provision and start a client.
+# Provision and start a client of ag-solo
```
Comment (dckc) [inline packages/cosmic-swingset/Makefile]:
```suggestion
# Provision and start an ag-solo client
```

ag-solo _is_ the client. It's a client _of_ the chain.

File: packages/cosmic-swingset/Makefile [hunk 2]
```diff
@@ -212,35 +212,23 @@ t1-provision-one-with-powers: wait-for-cosmos
...
 
-# Send some USDC to the vbank/provision module account where it can be traded for IST.
-fund-provision-pool:
-	$(MAKE) ACCT_ADDR=agoric1megzytg65cyrgzs6fvzxgrcqvwwl7ugpt62346 SOLO_COINS=1234000000ibc/usdc1234 fund-acct
-
 fund-acct: wait-for-cosmos
 	$(AGCH) \
 	  --home=t1/bootstrap --keyring-backend=test --from=bootstrap \
-	  tx bank send bootstrap $(ACCT_ADDR) $(SOLO_COINS) \
+	  tx bank send bootstrap $(ACCT_ADDR) $(FUNDS) \
```
Comment (dckc) [inline packages/cosmic-swingset/Makefile:218]: No default for `FUNDS`? Hm.

File: packages/wallet/ui/README.md [hunk 3]
```diff
@@ -24,9 +40,22 @@ open http://localhost:3000/wallet/
...
+https://psm.inter.trade/
+
+## Testing
+
+export AGORIC_NET=ollinet
+
+# make sure your wallet is provisioned on the net.
+agoric wallet list | grep KEPLR_WALLET_ADDR
+# if it's not in the list, provision it
+https://ollinet.faucet.agoric.net/
```
Comment (dckc) [inline packages/wallet/ui/README.md]: do we want ollinet URLs in our source code? I expect it to be a temporary service.

File: packages/wallet/ui/README.md [hunk 4]
```diff
@@ -24,9 +40,22 @@ open http://localhost:3000/wallet/
...
-make ACCT_ADDR=$KEPLR_ADDRESS AGORIC_POWERS=SMART_WALLET fund-acct provision-acct
-# now refresh the wallet UI and purses should load
+https://psm.inter.trade/
+
+## Testing
+
+export AGORIC_NET=ollinet
+
+# make sure your wallet is provisioned on the net.
+agoric wallet list | grep KEPLR_WALLET_ADDR
```
Comment (dckc) [inline packages/wallet/ui/README.md]: `agoric wallet show` seems more straightforward than `agoric wallet list | grep`. too noisy, I guess?

**Review by dckc — APPROVED** [review]
onward

File: packages/wallet/ui/README.md (hunk 4 above)
Comment (turadg) [reply packages/wallet/ui/README.md]: yeah, I was looking for just a little output. I was even thinking to make it `|| ` but thought that was too fancy

**Comment by turadg** [comment]
Changed landed in other PRS

**Pull request closed without merging**